ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -587,6 +587,28 @@ tests:
       ASSISTED_CONFIG: |
         IPXE_BOOT=true
     workflow: assisted-baremetal
+- as: e2e-metal-assisted-reclaim-single-node
```

Comment:
seems like the convention here is to have the prefix ``e2e-metal-assisted-kube-api``
```suggestion
- as: e2e-metal-assisted-kube-api-reclaim-single-node
```

Reply:
Good catch! Modified it to match, thank you!